CHANGELOG.md

```diff
@@ -4,6 +4,55 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 2019.02.0
+
+### Added
+- Classes needed by DQN: epsilon greedy strategy, discrete MLP model and
+  discrete qf-derived policy
+- Base Model class
+- Dump plots generated with Matplotlib in TensorBoard
+- Relative Entropy Policy Search (REPS) class
+- GaussianConvBaseline and GaussianConvRegressor class
+- Dockerfiles for garage
```

Comment:
New Dockerfiles, docker-compose files, and Makefiles for running garage using Docker